File: Iniciante/1114 - Senha Fixa/JavaScript/1114.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const CORRECT_PASSWORD = 2002;
  while (true) {
    const password = parseInt(lines.shift());
    if (password === CORRECT_PASSWORD) {
      console.log("Acesso Permitido");
      break;
    } else {
      console.log("Senha Invalida");
    }
  }
}

main();

File: Iniciante/1114 - Senha Fixa/Python/1114.py
def main():
    CORRECT_PASSWORD = 2002
    while True:
      password = int(input())
      if password == CORRECT_PASSWORD:
         print("Acesso Permitido")
         break
      else:
         print("Senha Invalida")


if __name__ == "__main__":
   main()
